.github/workflows/docker-build.yml

```
@@ -2,54 +2,22 @@ name: docker-build
-  #   - name: Rollout Restart Deployment
-  #     run: kubectl rollout restart deployment flask-test --kubeconfig kubeconfig
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        
+      - name: Checkout code
+        uses: actions/checkout@v4
+        
+      - name: Build Docker image
+        uses: docker/build-push-action@v5
+        with:
+          push: false
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)